foreman.fc

```diff
@@ -35,13 +35,16 @@
 
 /usr/lib/ruby/gems/1.8/gems/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
 /usr/lib64/ruby/site_ruby/1.8/x86_64-linux/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/usr/libexec/passenger/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
+
 /usr/lib/ruby/gems/1.8/gems/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/usr/share/passenger/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
+
 /var/run/rubygem-passenger(.*)?         gen_context(system_u:object_r:passenger_var_run_t,s0)
 
-# Passenger SCL file contexts
+# Passenger SCL file contexts (/opt/*/root prefix is added in the SCL meta package)
 
-/opt/theforeman/tfm/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
-/opt/theforeman/tfm/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/usr/(share/gems/gems|lib64/gems/exts|lib64/gems/ruby)/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
```

Comment:
Okay I replaced all three `lib64` with `lib`. Shell I also replace `lib64` with `lib*` in the relabel script, just for case?

I don't have time to setup testing environment for RHEL6 at the moment, so this is untested.